src/Controls/src/Core/Interactivity/EventTrigger.cs

```diff
@@ -71,7 +70,7 @@ void AttachHandlerTo(BindableObject bindable)
 			try
 			{
 				_eventinfo = bindable.GetType().GetRuntimeEvent(Event);
-				_handlerdelegate = s_handlerinfo.CreateDelegate(_eventinfo.EventHandlerType, this);
+				_handlerdelegate = ((EventHandler)OnEventTriggered).Method.CreateDelegate(_eventinfo.EventHandlerType, this);
```

Comment:
Reading this class, trying to undestand why so much System.Reflection usage in the original code...

I was wondering why it is not:
```csharp
EventHandler _handlerdelegate;
//...
_handlerdelegate = OnEventTriggered;
```

But I guess what they are doing here is trying to make `EventHandler<T>` of any type work?